libs/remix-ui/grid-view/src/lib/remix-ui-grid-cell.tsx

```diff
@@ -16,67 +16,97 @@ interface RemixUIGridCellProps {
   pinned?: boolean
   pinStateCallback?: any
   logo?: string
-  title?: string
+  title: string
   tagList?: string[] // max 8, others will be ignored
   classList?: string
   styleList?: any
   children?: ReactNode
+  expandViewEl?: any
+  handleExpand?: any
 }
 
 export const RemixUIGridCell = (props: RemixUIGridCellProps) => {
   const filterCon = useContext(FiltersContext)
   const [anyEnabled, setAnyEnabled] = useState(false)
+  const [expand, setExpand] = useState(false)
   const [pinned, setPinned] = useState<boolean>(props.pinned)
 
   useEffect(() => {
     if (props.tagList) setAnyEnabled(props.tagList.some((key) => filterCon.keyValueMap[key]?.enabled))
-    else setAnyEnabled(filterCon.showUntagged)
+    else setAnyEnabled(filterCon?.keyValueMap['no tag']?.enabled)
+    if (filterCon.filter != '') setAnyEnabled(anyEnabled && props.title.toLowerCase().includes(filterCon.filter))
+    console.log("pin ", pinned)
+
   }, [filterCon, props.tagList])
 
+  /*const listenOnExpand = (key) => {
```

Comment:
you should probably remove if it's unused.

Reply:
I will use this for next iteration. It's disabled but flags are there.